1.7β1 kops get ig returns SIGSEGV with get ig on darwin

When using kops 1.7 beta 1 to get ig information I received a segfault with a nil pointer reference.  

Darwin ITHAA1CSELLERS 16.6.0 Darwin Kernel Version 16.6.0: Fri Apr 14 16:21:16 PDT 2017; root:xnu-3789.60.24~6/RELEASE_X86_64 x86_64

`brew install --HEAD kops`

exception:
$ kops get ig
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x1db6252]

goroutine 1 [running]:
k8s.io/kops/pkg/client/simple/vfsclientset.(*VFSClientset).InstanceGroupsFor(0xc420019d90, 0x0, 0x2b, 0x0)
	/private/tmp/kops-20170720-86340-li679e/src/k8s.io/kops/pkg/client/simple/vfsclientset/clientset.go:54 +0x22
main.RunGetInstanceGroups(0xc420056478, 0xc420a21300, 0x0, 0x4, 0x1, 0xc420056440)
	/private/tmp/kops-20170720-86340-li679e/src/k8s.io/kops/cmd/kops/get_instancegroups.go:93 +0x234
main.NewCmdGetInstanceGroups.func1(0xc420225d40, 0xc420a21300, 0x0, 0x4)
	/private/tmp/kops-20170720-86340-li679e/src/k8s.io/kops/cmd/kops/get_instancegroups.go:65 +0x48
k8s.io/kops/vendor/github.com/spf13/cobra.(*Command).execute(0xc420225d40, 0xc420a212c0, 0x4, 0x4, 0xc420225d40, 0xc420a212c0)
	/private/tmp/kops-20170720-86340-li679e/src/k8s.io/kops/vendor/github.com/spf13/cobra/command.go:603 +0x22b
k8s.io/kops/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x5a81820, 0x3884300, 0x0, 0x0)
	/private/tmp/kops-20170720-86340-li679e/src/k8s.io/kops/vendor/github.com/spf13/cobra/command.go:689 +0x339
k8s.io/kops/vendor/github.com/spf13/cobra.(*Command).Execute(0x5a81820, 0x5ab1c70, 0x0)
	/private/tmp/kops-20170720-86340-li679e/src/k8s.io/kops/vendor/github.com/spf13/cobra/command.go:648 +0x2b
main.Execute()
	/private/tmp/kops-20170720-86340-li679e/src/k8s.io/kops/cmd/kops/root.go:98 +0x9b
main.main()
	/private/tmp/kops-20170720-86340-li679e/src/k8s.io/kops/cmd/kops/main.go:25 +0x2_0


Command should return (if proper environment settings are set e.g. --state --name):

kops get ig
NAME			ROLE	MACHINETYPE	MIN	MAX	SUBNETS
...
...
...

--- a/cmd/kops/create.go
+++ b/cmd/kops/create.go
@@ -176,6 +176,10 @@ func RunCreate(f *util.Factory, out io.Writer, c *CreateOptions) error {
 					return fmt.Errorf("error querying cluster %q: %v", clusterName, err)
 				}
 
+				if cluster == nil {
+					return fmt.Errorf("cluster %q not found", clusterName)
+				}
+
 				_, err = clientset.InstanceGroupsFor(cluster).Create(v)
 				if err != nil {
 					if apierrors.IsAlreadyExists(err) {

--- a/cmd/kops/get_instancegroups.go
+++ b/cmd/kops/get_instancegroups.go
@@ -90,6 +90,10 @@ func RunGetInstanceGroups(options *GetInstanceGroupsOptions, args []string) erro
 		return fmt.Errorf("error fetching cluster %q: %v", clusterName, err)
 	}
 
+	if cluster == nil {
+		return fmt.Errorf("cluster %q was not found", clusterName)
+	}
+
 	list, err := clientset.InstanceGroupsFor(cluster).List(metav1.ListOptions{})
 	if err != nil {
 		return err

--- a/cmd/kops/replace.go
+++ b/cmd/kops/replace.go
@@ -120,6 +120,9 @@ func RunReplace(f *util.Factory, cmd *cobra.Command, out io.Writer, c *ReplaceOp
 				if err != nil {
 					return fmt.Errorf("error fetching cluster %q: %v", clusterName, err)
 				}
+				if cluster == nil {
+					return fmt.Errorf("cluster %q not found", clusterName)
+				}
 				_, err = clientset.InstanceGroupsFor(cluster).Update(v)
 				if err != nil {
 					return fmt.Errorf("error replacing instanceGroup: %v", err)

--- a/cmd/kops/toolbox_convert_imported.go
+++ b/cmd/kops/toolbox_convert_imported.go
@@ -104,6 +104,10 @@ func RunToolboxConvertImported(f *util.Factory, out io.Writer, options *ToolboxC
 		return err
 	}
 
+	if cluster == nil {
+		return fmt.Errorf("cluster %q not found", options.ClusterName)
+	}
+
 	list, err := clientset.InstanceGroupsFor(cluster).List(metav1.ListOptions{})
 	if err != nil {
 		return err

--- a/federation/apply_federation.go
+++ b/federation/apply_federation.go
@@ -71,6 +71,9 @@ func (o *ApplyFederationOperation) FindKubecfg() (*kubeconfig.KubeconfigBuilder,
 		if err != nil {
 			return nil, fmt.Errorf("error reading cluster %q: %v", controller, err)
 		}
+		if cluster == nil {
+			return nil, fmt.Errorf("cluster %q not found", controller)
+		}
 
 		context, err := o.federationContextForCluster(cluster)
 		if err != nil {
@@ -120,6 +123,9 @@ func (o *ApplyFederationOperation) Run() error {
 		if err != nil {
 			return fmt.Errorf("error reading cluster %q: %v", controller, err)
 		}
+		if cluster == nil {
+			return fmt.Errorf("cluster %q not found", controller)
+		}
 
 		context, err := o.federationContextForCluster(cluster)
 		if err != nil {
@@ -153,6 +159,9 @@ func (o *ApplyFederationOperation) Run() error {
 		cluster, err := o.KopsClient.GetCluster(member)
 		if err != nil {
 			return fmt.Errorf("error reading cluster %q: %v", member, err)
+		}
+		if cluster == nil {
+			return fmt.Errorf("cluster %q not found", member)
 		}
 
 		clusterName := strings.Replace(cluster.Name, ".", "-", -1)

